### ISSUE TITLE
kotlin

### ISSUE THREAD

**Issue by imanoracle** [Jire/Charlatano#494]
Hi there,
Thank you for your efforts, when I run Charlatano, I get the following warning. How I can fix it?
WARN: [kts] kotlin.script.experimental.dependencies.DependenciesResolver.NoDepen
dencies must have a constructor without required parameters

**Comment by COLATHOS** [comment]
It does not matter. Just ignore it. No problem

**Comment by Zyphix** [comment]
I had the same problem you have to be on windows 10, and if it doesn’t work run it as admin that’s what I have to do.

**Comment by Zyphix** [comment]
Also make sure you’re in a game when u start it, if you start before sometimes it doesn’t work or crashes your game

**Comment by imanoracle** [comment]
@Zyphix Oh, then first I should start the game with Steam, then start Charlatano? I thought it is vice versa!

**Comment by Zyphix** [comment]
Yes @imanoracle make sure you’re in a game holding a weapon or sometimes it won’t work.

**Comment by woodygamer** [comment]
Can i use Charlatano 1.0.6 in MM is it VAC undetected?
